python/fasttrackml/_tracking_service/client.py

```diff
@@ -0,0 +1,29 @@
+from fasttrackml.entities.metric import Metric
+from fasttrackml.store.custom_rest_store import CustomRestStore
+from mlflow.tracking._tracking_service.client import TrackingServiceClient
+from mlflow.tracking.metric_value_conversion_utils import (
+    convert_metric_value_to_float_if_possible,
+)
+from mlflow.utils import chunk_list
+from mlflow.utils.rest_utils import MlflowHostCreds
+from mlflow.utils.time import get_current_time_millis
+from mlflow.utils.validation import MAX_METRICS_PER_BATCH
+
+
+class FasttrackmlTrackingServiceClient(TrackingServiceClient):
+
+    def __init__(self, tracking_uri):
+        super().__init__(tracking_uri)
+        self.store_with_context = CustomRestStore(lambda: MlflowHostCreds(self.tracking_uri))
+
+    def log_metric(self, run_id, key, value, timestamp=None, step=None, context=None):
```

Comment:
should we have type hints for the attributes?

Reply:
We do have type hints in fluent.py, here I didn't use type hints just because in the original implementation in MLflow there aren't but if you think it's better to have theme also here I think we could just add them.

Reply:
yes, I think they're worth adding